packages/patternlab/styleguide/source/assets/js/modules/headerSearch.js

```diff
@@ -61,37 +61,32 @@ export default (function (window, document,$) {
                 }
             }, 200);
         };
-        $searchInput.on('blur', function() {
+        $searchInput.on("blur", function() {
             handleBlur();
-        })
-        $options.on('blur', function() {
+        });
+        $options.on("blur", function() {
             handleBlur();
-        })
+        });
         // when the input is in focus and has a value, show the suggestions
-        $searchInput.on('focus', function() {
+        $searchInput.on("focus", function() {
             const inputValue = $(this).val();
             if(inputValue) {
                 openDropdown();
             }
-        })
+        });
 
         // keydown from input move focus into the suggestion options
         $searchInput.keydown(function(e) {
             if (key(e).arrowDown) {
                 focusIndex = 0;
                 setFocus(focusIndex);
             }
-            if (key(e).arrowUp) { 
+            if (key(e).arrowUp) {
                 focusIndex = [optionsCount -1];
                 setFocus(focusIndex);
             }
         });
-              
 
-        // transform org name to value
-        const slugify = (string = '') => (
```

Comment:
The org values needs to be slugified - this is based on how the Google search parameter values are tokenized. 
Before: https://search.mass.gov/?q=tax&org=department-of-conservation--recreation

![Screenshot 2025-04-28 at 12 21 23 AM](https://github.com/user-attachments/assets/364b02da-3cb4-47a7-a056-09f08348eede)


After: https://search.mass.gov/?q=tax&org=Department+of+Conservation+%26+Recreation (Google returns no results and search filter tag is empty)

![Screenshot 2025-04-28 at 12 21 55 AM](https://github.com/user-attachments/assets/eaa91ac5-19e6-45c2-9809-654a55fc0996)

Based on the new data structure, this function probably should be moved into the openmass codebase, the value should be passed as slugified (see code example in the data object)

Reply:
Correct, we're slugifying that value server side in this paired PR

https://github.com/massgov/openmass/pull/2944/files#diff-7ab1b4b71e92e70b91efbdb1fbe44676f793a4eab00ac5d1269180e1f0bb3e14R1313